data/README.md

```diff
@@ -44,6 +44,7 @@
 | rle-dict-snappy-checksum.parquet                 | compressed and dictionary-encoded INT32 and STRING columns in format v2 with a matching CRC |
 | plain-dict-uncompressed-checksum.parquet         | uncompressed and dictionary-encoded INT32 and STRING columns in format v1 with a matching CRC |
 | rle-dict-uncompressed-corrupt-checksum.parquet   | uncompressed and dictionary-encoded INT32 and STRING columns in format v2 with a mismatching CRC |
+| chunked_string_map.parquet             | Map(String, int32) containing string that won't fit arrow Binary. Asserts arrow LargeBinary can read it [Issue](https://github.com/apache/arrow/issues/32723) |
```

Comment:
Some thoughts:
- File name could be more clear, like `large_string_map.brotli.parquet`.
- The link of github issue may be invalid in the future. What about adding a separate md file to include its generation script, file metadata (optional if the script is clear enough) and explain this issue in more detail?
- arrow Binary -> arrow BinaryArray
- arrow LargeBinary -> arrow LargeBinaryArray
- Have you tried other codec like gzip (and higher levels)? It may help further reduce the file size.

Reply:
Regarding #2, there are some other files in this list that point to JIRA issues. This GH issue has an equivalent JIRA issue, maybe I can point to the JIRA one instead?